content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst

```diff
@@ -2,62 +2,60 @@
 Integrating additional costs to products (landed costs)
 =======================================================
 
-The landed cost feature in Odoo allows to include additional costs
-(shipment, insurance, customs duties, etc.) into the cost of the
-product.
-
-.. note::
-         Landed costs can only be applied to products with a FIFO or AVCO costing
-         method and an automated inventory valuation (which requires the
-         accounting application to be installed).
+The landed cost feature in Odoo allows to include additional costs (shipment, insurance, customs
+duties, etc.) into the cost of the product.
 
 Configuration
 =============
 
-First, you need to go in :menuselection:`Inventory --> Configuration --> Settings` and
-activate the landed costs feature. You can also determine the default
-journal in which the landed cost accounting entries will be recorded.
+First, you need to go in :menuselection:`Inventory --> Configuration --> Settings` and activate the
+landed costs feature. You can also determine the default journal in which the landed cost
+accounting entries will be recorded.
 
-.. image:: integrating_landed_costs/integrating_landed_costs_01.png
-    :align: center
+.. image:: integrating_landed_costs/landed-costs-setting.png
+   :align: center
+   :alt: Activate the landed cost feature in Inventory Settings
 
 Add costs to products
 =====================
 
 Receive the vendor bill
 -----------------------
 
-Let’s imagine I receive a bill from custom duties for a shipment. I’ll
-tick the box *Landed Costs* on the vendor bill line.
+Let's imagine I receive a bill from custom duties for a shipment. I'll tick the box *Landed Costs*
+on the vendor bill line.
 
-.. image:: integrating_landed_costs/integrating_landed_costs_02.png
-    :align: center
+.. image:: integrating_landed_costs/landed-costs-field-vendor-bill.png
+   :align: center
+   :alt: Enable Landed Costs field on vendor bill line.
 
 .. note::
-         The landed cost product must be of type service.
+   The landed cost product must be of type service.
 
-If this product is always a landed cost, you can also define it on the
-product and avoid having to tick the box on each vendor bill.
+If this product is always a landed cost, you can also define it on the product and avoid having to
+tick the box on each vendor bill.
 
-.. image:: integrating_landed_costs/integrating_landed_costs_03.png
-    :align: center
+.. image:: integrating_landed_costs/define-landed-cost-on-product.png
+   :align: center
+   :alt: Option to define a product as always a landed cost
 
-At the top of my vendor bill, I’ll see a button *create landed costs*.
-I click on this button and a landed cost is automatically created. I can
-now decide on which picking those additional costs should apply.
+At the top of my vendor bill, I'll see a button *create landed costs*. I click on this button and a
```

Comment:
Replace the three sentences on lines 42-44 `At the top...costs should apply.` with:

``Once the landed cost is added to the vendor bill (either by checking the :guilabel:`Landed Cost` option on the invoice line or adding a landed cost product to the bill), click the :guilabel:`Create Landed Costs` button at the top of the bill. Odoo automatically creates a landed cost record with the set landed cost pre-filled in the :guilabel:`Additional Costs` product lines. From here, decide which picking the additional costs apply to by clicking :guilabel:`Edit` and selecting the picking reference number from the :guilabel:`Transfers` drop-down menu. Finally, click :guilabel:`Save`.``